### DISCUSSION TITLE
Ability to change morph target values of GLTF

### DISCUSSION THREAD

**Discussion by squinonescolon** [cables-gl/cables_docs#279] [Feature Requests]
After importing a GLTF model with morph targets into Cables I would like to be able to either randomly or deliberately chose a different influence value for one or more of the morph targets to vary and display new shapes of the model whenever I desired. I think [#285](https://github.com/cables-gl/cables_docs/discussions/285) is tangentially related.

**Comment by tastez** [comment]
some xtra info- 
i believe this is called Relative shape key blending in Blender. When you play back a GLTF with keyframed Shape Keys in Don McCurdy's GLTF viewer the shape keys will be shown as Morph Targets in the sidepanel

**Reply by squinonescolon** [reply]
You're correct, I'm changing the title to reflect the subject a little better.

**Reply by tastez** [reply]
whats not clear to me is the state of the GLTF players in webgl / Blender export compatibility. For example..are actions with keyframed linear shape key influence supported (on some gltf viewers i think not), are Shape Key editor keys the same as actions (i dont think so..) . It's pretty confusing but relative playback would be sweet

**Reply by squinonescolon** [reply]
The terminology is very confusing and I'm an artist not a programmer. I just know enough to be dangerous ;-). I'm currently using Babylonjs to do this very thing and it's pretty easy. I would much rather use Cables for this but I can't figure out how to do it except for modifying GltfScene_v3 and that would be a tall order for me.

**Reply by squinonescolon** [reply]
And if I remember correctly Blender Shape key editor values map to Morph target influence number and the Blender action itself is what players seem to think is the "animation".
